docker/compose/README.md

```diff
@@ -5,27 +5,32 @@ custom_edit_url: https://github.com/diem/diem/edit/master/docker/compose/README.
 ---
 
 This directory contains the following compose configurations:
-* **validator-testnet**: creates a single validator test network
-* **faucet**: creates a faucet that directly connects to validator-testnet
+* **validator-testnet**: creates a single validator test network, and a faucet that directly connects to it
+* **client-cli**: creates a CLI client which connects to the above validator/faucet
```

Comment:
it seems this rename triggers a lot renaming :(, if you want to keep original "client", I'm fine with it.

Reply:
wasn't too bad, I agree it provides clarity :-)